### ISSUE TITLE
file is unwritable

### ISSUE THREAD

**Issue by Anveshaaaa** [naztronaut/dancyPi-audio-reactive-led#72]
we want to change the card value, but keep getting the error that the file (sc_asound.conf) is unwritable. Does anyone know how to fix this?

**Comment by Anveshaaaa** [comment]
yes thank you, we are able to change it now! :) But we have another issue, when we type 'arecord -l' we don't see a list of capture hardware devices and when we type 'aplay -l' we don't see loopback devices

**Comment by Anveshaaaa** [comment]
aisha@mypi:~ $ aplay -l
**** List of PLAYBACK Hardware Devices ****
card 0: Headphones [bcm2835 Headphones], device 0: bcm2835 Headphones [bcm2835 Headphones]
  Subdevices: 8/8
  Subdevice #0: subdevice #0
  Subdevice #1: subdevice #1
  Subdevice #2: subdevice #2
  Subdevice #3: subdevice #3
  Subdevice #4: subdevice #4
  Subdevice #5: subdevice #5
  Subdevice #6: subdevice #6
  Subdevice #7: subdevice #7
card 1: vc4hdmi0 [vc4-hdmi-0], device 0: MAI PCM i2s-hifi-0 [MAI PCM i2s-hifi-0]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 2: vc4hdmi1 [vc4-hdmi-1], device 0: MAI PCM i2s-hifi-0 [MAI PCM i2s-hifi-0]
  Subdevices: 1/1
  Subdevice #0: subdevice #0
aisha@mypi:~ $ arecord -l
**** List of CAPTURE Hardware Devices ****

**Comment by Randyclaus** [comment]
I am having this same issue.  I am also getting errors when trying to install numpy.
